### DISCUSSION TITLE
some problems (icons and mason-lspconfig.nvim)

### DISCUSSION THREAD

**Discussion by Moa-C8** [LazyVim/LazyVim#353] [General]
Hi, I am a beginner in linux and even more with nvim/vim I tried to install lazyvim but I have some problems not important

- I have no icons in the menu of lazyvim and in the command line.

- every time I load a file I get 2 msgboxes and they don't match 

1. [mason-lspconfig.nvim] jsonls installation
2. [mason-lspconfig.nvim] failed to install jsonls. The installation logs are available in :Mason and :MasonLOG (but I can't find the Mason and MasonLOG).

It still works fine but I want to fix it if someone can help me.

**Comment by dpetka2001** [comment]
For the icons you would have to install a patched font that supports this. Many people use [Nerdfonts](https://www.nerdfonts.com/). If you install a font of your preference from the site, you should be able to see the icons. For your second problem, there does not exist a `jsonls` in Mason. This exists in nvim-lspconfig only. The corresponding mapping in mason registry is `json-lsp`, so you should try this if you want to install it with `Mason`.

**Reply by folke** [reply]
I believe mason-lspconfig, will map jsonls to json-lsp.

To get `MasonLog`, execute `:Mason` first, then quit the dialog and run `:MasonLog`

**Reply by dpetka2001** [reply]
As far as my little understanding goes, mason-lspconfig exists for bridging the gap to setup the servers installed by mason with nvim-lspconfig. But if you have in mason `ensure_installed` a server that cannot find in its registry (for example `jsonls`) it will throw an error not being able to find the package in mason-registry. It has to be the mapping that mason expects to search for in its registry. I just tried in mason `ensure_installed` with `jsonls` and throws an error about not being able to find the package, whereas `json-lsp` works as expected.

**Reply by dpetka2001** [reply]
Of course if you put in nvim-lspconfig `servers` the server `jsonls` it  works as expected. It depends where you define the server to be installed I believe.

**Reply by folke** [reply]
yep, that's how it works indeed, so that may indeed be OP's issue

**Reply by Moa-C8** [reply]
thank you i just ran :Mason and i found the problem i had not installed node js
But I have another problem in :Mason failed flake8 unable to create python3 venv environement but I have already installed flake8 and virtualenv. it sends me back to pip3/init.lua line 90 I went to see and in the file it uses the command with python but on my distribution the only command that works is python3 I have to reinstall python otherwise or change somehing ?

**Reply by folke** [reply]
no idea. Maybe run `:checkhealth mason` and if that doesn't show you what's wrong, you can check the mason issues or post a new one

**Reply by Moa-C8** [reply]
ok ty after a restart of my computer I have no more problem
